liferay-echo/liferay-portal
# LPS-191073 Fix simulation iframe reload after simulating experiences

Motivation
=======
After simulating experiences in the simulation panel, simulating segments does not work.
[Link to story or ticket](https://liferay.atlassian.net/browse/LPS-191073)

Proposed Solution
========
For updating the iframe content now we use the iframe src attribute.

Steps to Verify:
----------------
1. Add the feature.flag.LPS-186558=true into portal-ext.properties
2. Add a segment
3. In the home page, add an experience of that segment
4. Add a Button fragment for that experience
5. Publish the content page
6. Go to Simulation panel
7. Simulate the 2 segments that we have
8. Switch to simulate by experiences and simulate the two experiences
9. Select simulation by segments and switch between the two segments and check that everything keeps working fine.
